# Univention Virtual Machine Manager Daemon.
#
# Copyright 2010 Univention GmbH
#
# http://www.univention.de/
#
# All rights reserved.
#
# The source code of the software contained in this package
# as well as the source package itself are made available
# under the terms of the GNU Affero General Public License version 3
# (GNU AGPL V3) as published by the Free Software Foundation.
#
# Binary versions of this package provided by Univention to you as
# well as other copyrighted, protected or trademarked materials like
# Logos, graphics, fonts, specific documentations and configurations,
# cryptographic keys etc. are subject to a license agreement between
# you and Univention.
#
# In the case you use the software under the terms of the GNU AGPL V3,
# the program is provided in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU Affero General Public License for more details.
#
# You should have received a copy of the GNU Affero General Public
# License with the Debian GNU/Linux or Univention distribution in file
# /usr/share/common-licenses/AGPL-3; if not, see
# <http://www.gnu.org/licenses/>.
#
# Philipp Hahn <hahn@univention.de>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version: univention-virtual-machine-manager 0.9.10-1\n"
"Report-Msgid-Bugs-To: packages@univention.de\n"
"POT-Creation-Date: 2011-04-21 11:10+0200\n"
"PO-Revision-Date: 2011-04-21 10:25+0200\n"
"Last-Translator: Philipp Hahn <hahn@univention.de>\n"
"Language-Team: German <de@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: src/univention-virtual-machine-manager-daemon:55
#: src/univention-virtual-machine-manager-daemon:65
#: src/univention-virtual-machine-manager-daemon:122
#, python-format
msgid "%(strerror)s [%(errno)d]"
msgstr "%(strerror)s [%(errno)d]"

#: src/univention/uvmm/unix.py:257
msgid "Authentication failed"
msgstr "Authentifizierung fehlgeschlagen"

#: src/univention/uvmm/unix.py:287
msgid "Authentication protocol violated"
msgstr "Authentifizierungsprotokoll verletzt"

#: src/univention/uvmm/commands.py:356
#, python-format
msgid "Command \"%(command)s\" is restricted"
msgstr "Kommando \"%(command)s\" ist verboten"

#: src/univention-virtual-machine-manager-daemon:183
msgid "Connection options"
msgstr "Verbindungsoptionen"

#: src/univention/uvmm/client.py:166 src/univention/uvmm/client.py:191
#, python-format
msgid "Could not connect to \"%(host)s:%(port)d\": %(errno)d"
msgstr "Keine Verbindung zu \"%(host)s:%(port)d\": %(errno)d"

#: src/univention/uvmm/uvmm_ldap.py:186
msgid "Could not open LDAP-Admin connection"
msgstr "Keine LDAP-Verbindung als Administrator"

#: src/univention/uvmm/uvmm_ldap.py:168
msgid "Could not open LDAP-Machine connection"
msgstr "Keine LDAP-Verbindung als Host"

#: src/univention/uvmm/client.py:116
#, python-format
msgid "Could not open socket \"%(path)s\": %(errno)d"
msgstr "Konnte Socket \"%(path)s\" nicht öffnen: %(errno)d"

#: src/univention/uvmm/uvmm_ldap.py:160
#, python-format
msgid "Could not query \"%(uri)s\""
msgstr "Keine Antwort für \"%(uri)s\""

#: src/univention/uvmm/client.py:68
#, python-format
msgid "Could not send request: %(errno)d"
msgstr "Konnte Socket nicht öffnen: %(errno)d"

#: src/univention-virtual-machine-manager-daemon:201
msgid "Daemon options"
msgstr "Dämonoptionen"

#: src/univention/uvmm/node.py:1235
#, python-format
msgid "Domain \"%(domain)s\" in state \"%(state)s\" can not be migrated"
msgstr ""
"Domain \"%(domain)s\" im Zustand \"%(state)s\" kann nicht migriert werden"

#: src/univention/uvmm/node.py:1009
#, python-format
msgid "Domain name \"%(domain)s\" already used by \"%(uuid)s\""
msgstr "Domain-Name \"%(domain)s\" wird bereits von \"%(uuid)s\" verwendet"

#: src/univention/uvmm/client.py:97
msgid "EOS while waiting for answer."
msgstr "Ende des Datenstroms beim Warten auf eine Antwort"

#: src/univention/uvmm/node.py:1273
#, python-format
msgid "Error creating \"%(domain)s\" snapshot: %(error)s"
msgstr "Fehler beim Sichern der Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/storage.py:74
#, python-format
msgid "Error creating storage pool \"%(pool)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Fehler beim Erzeugen des Speicherbereichs \"%(pool)s\" für Domäne \"%(domain)"
"s\": %(error)s"

#: src/univention/uvmm/storage.py:188
#, python-format
msgid ""
"Error creating storage volume \"%(name)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Fehler beim Erzeugen des Speichermediums \"%(name)s\" für \"%(domain)s\": %"
"(error)s"

#: src/univention/uvmm/node.py:1066
#, python-format
msgid "Error defining domain \"%(domain)s\": %(error)s"
msgstr "Fehler beim Definieren der Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/node.py:1313 src/univention/uvmm/node.py:1322
#, python-format
msgid "Error deleting \"%(domain)s\" snapshot: %(error)s"
msgstr ""
"Fehler beim Löschen des Sciherungsstandes von \"%(domain)s\": %(error)s"

#: src/univention/uvmm/storage.py:199 src/univention/uvmm/storage.py:292
#, python-format
msgid "Error listing pools at \"%(uri)s\": %(error)s"
msgstr "Fehler beim Anzeigen der Speicherbereiche von \"%(uri)s\": %(error)s"

#: src/univention/uvmm/storage.py:104
#, python-format
msgid "Error locating storage pool \"%(pool)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Fehler beim Dinder des Speicherbereiche \"%(pool)s\" von \"%(domain)s\": %"
"(error)s"

#: src/univention/uvmm/storage.py:86 src/univention/uvmm/storage.py:121
#, python-format
msgid ""
"Error locating storage volume \"%(volume)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Fehler beim Finden der Speichermediums \"%(volume)s\" von \"%(domain)s\": %"
"(error)s"

#: src/univention/uvmm/node.py:1153
#, python-format
msgid "Error managing domain \"%(domain)s\""
msgstr "Fehler beim Verwalten der Domäne \"%(domain)s\""

#: src/univention/uvmm/node.py:1156 src/univention/uvmm/node.py:1159
#, python-format
msgid "Error managing domain \"%(domain)s\": %(error)s"
msgstr "Fehler beim Verwalten der Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/node.py:1252
#, python-format
msgid "Error migrating domain \"%(domain)s\": %(error)s"
msgstr "Fehler beim Migrieren der Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/uvmm_ldap.py:125
#, python-format
msgid "Error reading %(file)s: %(msg)s"
msgstr "Fehler beim Lesen der Datei %(file)s: %(msg)s"

#: src/univention/uvmm/uvmm_ldap.py:128
#, python-format
msgid "Error reading incomplete %(file)s."
msgstr "Fehler beim Lesen der unvollständigen Datei %(file)s"

#: src/univention/uvmm/uvmm_ldap.py:101
#, python-format
msgid "Error removing %(file)s.old: %(msg)s"
msgstr "Fehler beim Entfernen der Datei %(file)s.old: %(msg)s"

#: src/univention/uvmm/node.py:1057
#, python-format
msgid "Error removing domain \"%(domain)s\": %(error)s"
msgstr "Fehler beim Löschen der Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/uvmm_ldap.py:111
#, python-format
msgid "Error renaming %(file)s.new: %(msg)s"
msgstr "Fehler beim Umbenennen der Datei %(file)s.new: %(msg)s"

#: src/univention/uvmm/uvmm_ldap.py:106
#, python-format
msgid "Error renaming %(file)s: %(msg)s"
msgstr "Fehler beim Umbenennen der Datei %(file)s: %(msg)s"

#: src/univention/uvmm/node.py:1187
#, python-format
msgid "Error restoring domain \"%(domain)s\": %(error)s"
msgstr "Fehler beim Wiederherstellen der Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/network.py:55
#, python-format
msgid "Error retrieving list of active networks: %(error)s"
msgstr "Fehler beim Auslesen der aktiven Netzwerke: %(error)s"

#: src/univention/uvmm/network.py:76
#, python-format
msgid "Error retrieving list of networks: %(error)s"
msgstr "Fehler beim Auslesen der Netzwerke: %(error)s"

#: src/univention/uvmm/node.py:1014 src/univention/uvmm/node.py:1023
#, python-format
msgid "Error retrieving old domain \"%(domain)s\": %(error)s"
msgstr "Fehler beim Auslesen der alten Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/node.py:1290 src/univention/uvmm/node.py:1296
#, python-format
msgid "Error reverting \"%(domain)s\" to snapshot: %(error)s"
msgstr "Fehler beim Wiederherstellen der Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/node.py:1173
#, python-format
msgid "Error saving domain \"%(domain)s\": %(error)s"
msgstr "Fehler beim Speichern der Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/network.py:69
#, python-format
msgid "Error starting network %(name)s: %(error)s"
msgstr "Fehler beim Starten des Netzwerks %(name)s: %(error)s"

#: src/univention/uvmm/node.py:1209
#, python-format
msgid "Error undefining domain \"%(domain)s\": %(error)s"
msgstr "Fehler beim Löschen der Domäne \"%(domain)s\": %(error)s"

#: src/univention/uvmm/node.py:1340 src/univention/uvmm/node.py:1357
#, python-format
msgid "Error updating domain \"%(domain)s\""
msgstr "Fehler beim Aktualisieren der Domäne \"%(domain)s\""

#: src/univention/uvmm/client.py:105
#, python-format
msgid "Error while closing socket: %(errno)d"
msgstr "Fehler beim Löschen des Sockets: %(errno)d"

#: src/univention/uvmm/client.py:95
#, python-format
msgid "Error while waiting for answer: %(errno)d"
msgstr "Fehler beim Warten auf eine Antwort: %(errno)d"

#: src/univention/uvmm/unix.py:160
#, python-format
msgid "Exception: %(exception)s"
msgstr "Fehler: %(exception)s"

#: src/univention-virtual-machine-manager-daemon:119
msgid "Failed to fork daemon."
msgstr "Dämon konnte sich nicht abspalten."

#: src/univention/uvmm/node.py:1085 src/univention/uvmm/node.py:1088
msgid "Failed to update the additionally information in the LDAP directory."
msgstr ""
"Die Aktualisierung zusätzlicher Informationen im LDAP-Verzeichnis ist "
"fehlgeschlagen."

#: src/univention/uvmm/node.py:1362
#, python-format
msgid "Failto to update domain \"%(domain)s\""
msgstr "Aktualisieren der Domäne \"%(domain)s\" fehlgeschlagen"

#: src/univention-virtual-machine-manager-daemon:207
msgid "Fork into background"
msgstr "Abspalten in den Hintergrund"

#: src/univention-virtual-machine-manager:566
#: src/univention-virtual-machine-manager-daemon:192
msgid "HOST and PORT number of the SSL encrypted TCP socket"
msgstr "RECHNER und PORT-Nummer für die SSL verschlüsselte TCP Verbindung"

#: src/univention-virtual-machine-manager:563
#: src/univention-virtual-machine-manager-daemon:189
msgid "HOST and PORT number of the TCP socket"
msgstr "RECHNER und PORT-Nummer für die TCP Verbindung"

#: src/univention-virtual-machine-manager:562
#: src/univention-virtual-machine-manager:565
#: src/univention-virtual-machine-manager-daemon:188
#: src/univention-virtual-machine-manager-daemon:191
msgid "HOST:PORT"
msgstr "RECHNER:PORT"

#: src/univention/uvmm/node.py:643
#, python-format
msgid "Hypervisor \"%(uri)s\" is already connected."
msgstr "Verbindung zum Hypervisor \"%(uri)s\" besteht bereits."

#: src/univention/uvmm/node.py:656 src/univention/uvmm/node.py:668
#, python-format
msgid "Hypervisor \"%(uri)s\" is not connected."
msgstr "Noch keine Verbindung zum Hypervisor \"%(uri)s\"."

#: src/univention/uvmm/node.py:665
#, python-format
msgid "Hypervisor \"%(uri)s\" is unavailable."
msgstr "Verbindung zum Hypervisor \"%(uri)s\" ist gestört."

#: src/univention/uvmm/protocol.py:77
#, python-format
msgid "Incompatible version: %(major)d.%(minor)d"
msgstr "Inkompatible Version: %(major)d.%(minor)d"

#: src/univention-virtual-machine-manager:198
msgid "Insufficient arguments"
msgstr "Fehlende Argumente"

#: src/univention-virtual-machine-manager-daemon:198
#, python-format
msgid "LDAP cache directory [%(default)s]"
msgstr "Verzeichnis für LDAP-Zwischenspeicher [%(default)s]"

#: src/univention-virtual-machine-manager-daemon:195
msgid "LDAP options"
msgstr "LDAP-Optionen"

#: src/univention/uvmm/uvmm_ldap.py:135
msgid "No SERVICES defined."
msgstr "Keine Dienste(SERVICES) definiert."

#: src/univention-virtual-machine-manager:581
msgid "No mode specified"
msgstr "Kein Modus angegeben"

#: src/univention-virtual-machine-manager:140
msgid "No nodes found."
msgstr "Keine Knoten gefunden."

#: src/univention/uvmm/commands.py:91
#, python-format
msgid "Node %(uri)s is not available"
msgstr "Knoten %(uri)s ist nicht verfügbar"

#: src/univention/uvmm/protocol.py:86
#, python-format
msgid "Not a Packet: %(type)s"
msgstr "Kein Paket: %(type)s"

#: src/univention/uvmm/client.py:89
msgid "Not a UVMM_Response."
msgstr "Keine UVMM-Antwort."

#: src/univention/uvmm/protocol.py:84
#, python-format
msgid "Not a valid Packet: %(msg)s"
msgstr "Kein gültiges Paket: %(msg)s"

#: src/univention/uvmm/client.py:146
#, python-format
msgid "PAM error: %(msg)s"
msgstr "PAM Fehler: %(msg)s"

#: src/univention/uvmm/client.py:148
#, python-format
msgid "PAM info: %(msg)s"
msgstr "PAM Information: %(msg)s"

#: src/univention/uvmm/unix.py:113
#, python-format
msgid "Packet is no UVMM Request: %(type)s"
msgstr "Paket ist keine UVMM-Anfrage: %(type)s"

#: src/univention-virtual-machine-manager:560
msgid "Path to the UNIX socket"
msgstr "Pfad zum UNIX-Socket"

#: src/univention-virtual-machine-manager-daemon:186
#, python-format
msgid "Path to the UNIX socket [%(default)s]"
msgstr "Pfad zum UNIX-Socket [%(default)s]"

#: src/univention-virtual-machine-manager-daemon:216
#, python-format
msgid "Path to the ini-file [%(default)s]"
msgstr "Pfad zur ini-Datei [%(default)s]"

#: src/univention-virtual-machine-manager-daemon:210
msgid "Path to the log-file"
msgstr "Pfad zur Log-Datei"

#: src/univention-virtual-machine-manager-daemon:204
#, python-format
msgid "Path to the pid-file [%(default)s]"
msgstr "Pfad zur pid-Datei [%(default)s]"

#: src/univention-virtual-machine-manager:569
#: src/univention-virtual-machine-manager-daemon:213
msgid "Print additional information"
msgstr "Ausführliche Informationen ausgeben"

#: src/univention-virtual-machine-manager-daemon:219
msgid "Skip reading ini-file"
msgstr "Lesen der ini-Datei überspringen"

#: src/univention/uvmm/node.py:1259 src/univention/uvmm/node.py:1264
#: src/univention/uvmm/node.py:1280 src/univention/uvmm/node.py:1285
#: src/univention/uvmm/node.py:1303 src/univention/uvmm/node.py:1308
#, python-format
msgid "Snapshot not supported \"%(node)s\""
msgstr "\"%(node)s\" unterstützt keine Sicherungspunkte."

#: src/univention/uvmm/client.py:150
#, python-format
msgid "Unknown PAM type: %(type)s"
msgstr "Unbekannter PAM-Typ: %(type)s"

#: src/univention/uvmm/commands.py:360
#, python-format
msgid "Unknown command \"%(command)s"
msgstr "Unbekannter Befehl \"%(command)s\""

#: src/univention/uvmm/node.py:350
#, python-format
msgid "Unknown disk type: %(type)d"
msgstr "Unbekannter Laufwerkstyp: %(type)d"

#: src/univention-virtual-machine-manager:191
#: src/univention-virtual-machine-manager:583
#, python-format
msgid "Unknown mode \"%(mode)s\""
msgstr "Unbekannter Modus \"%(mode)s\""

#: src/univention/uvmm/node.py:1127
#, python-format
msgid "Unsupported state transition %(cur_state)s to %(next_state)s"
msgstr "Ungültiger Zustandsübergang von %(cur_state)s zu %(next_state)s"

#: src/univention/uvmm/storage.py:175
#, python-format
msgid "Unsupported storage-pool-type \"%(pool_type)s for \"%(domain)s\""
msgstr ""
"Speicherbereichstyp \"%(pool_type)s\" wird nicht unterstützt für \"%(domain)s"
"\""

#: src/univention/uvmm/commands.py:153
#, python-format
msgid "definition != Domain: %(domain)s"
msgstr "definition != Domain: %(domain)s"

#: src/univention/uvmm/commands.py:170 src/univention/uvmm/commands.py:185
#: src/univention/uvmm/commands.py:200 src/univention/uvmm/commands.py:215
#: src/univention/uvmm/commands.py:236 src/univention/uvmm/commands.py:251
#: src/univention/uvmm/commands.py:266 src/univention/uvmm/commands.py:281
#: src/univention/uvmm/commands.py:294
#, python-format
msgid "domain != string: %(domain)s"
msgstr "domain != Zeichenkette: %(domain)s"

#: src/univention/uvmm/commands.py:121
#, python-format
msgid "group != string: %(group)s"
msgstr "group != Zeichenkette: %(group)s"

#: src/univention/uvmm/commands.py:108
#, python-format
msgid "hz != int: %(hz)s"
msgstr "hz != Ganzzahl: %(hz)s"

#: src/univention-virtual-machine-manager:557
msgid "show this help message and exit"
msgstr "Anzeigen dieses Hilfetextes und beenden"

#: src/univention/uvmm/commands.py:253 src/univention/uvmm/commands.py:268
#: src/univention/uvmm/commands.py:283
#, python-format
msgid "snapshot != string: %(snapshot)s"
msgstr "snapshot != Zeichenkette: %(snapshot)s"

#: src/univention/uvmm/commands.py:187 src/univention/uvmm/commands.py:202
#, python-format
msgid "statefile != string: %(file)s"
msgstr "statefile != Zeichenkette: %(file)s"

#: src/univention/uvmm/commands.py:238
#, python-format
msgid "target_uri != string: %(uri)s"
msgstr "target_uri != Zeichenkette: %(uri)s"

#: src/univention/uvmm/commands.py:89
#, python-format
msgid "unknown node %(uri)s"
msgstr "Unbekannter Knoten %(uri)s"

#: src/univention/uvmm/commands.py:172
#, python-format
msgid "unsupported state: %(state)s"
msgstr "Unbekannter Zustand: %(state)s"

#: src/univention/uvmm/commands.py:59 src/univention/uvmm/commands.py:71
#: src/univention/uvmm/commands.py:83 src/univention/uvmm/commands.py:110
#: src/univention/uvmm/commands.py:151 src/univention/uvmm/commands.py:168
#: src/univention/uvmm/commands.py:183 src/univention/uvmm/commands.py:198
#: src/univention/uvmm/commands.py:213 src/univention/uvmm/commands.py:234
#: src/univention/uvmm/commands.py:249 src/univention/uvmm/commands.py:264
#: src/univention/uvmm/commands.py:279 src/univention/uvmm/commands.py:305
#: src/univention/uvmm/commands.py:319 src/univention/uvmm/commands.py:335
#, python-format
msgid "uri != string: %(uri)s"
msgstr "uri != Zeichenkette: %(uri)s"

#: src/univention-virtual-machine-manager:552
#, python-format
msgid "usage: %%prog [options] {%(modes)s} uri"
msgstr "Aufruf: %%prog [Optionen] {%(modes)s} uri"

#: src/univention-virtual-machine-manager-daemon:181
msgid "usage: %prog [options] [uri...]"
msgstr "Aufruf: %prog [Optionen] [uri...]"

#: src/univention/uvmm/commands.py:217
#, python-format
msgid "volumes != list or None: %(volumes)s"
msgstr "volumes != Liste oder None: %(volumes)s"

#: src/univention/uvmm/commands.py:221 src/univention/uvmm/commands.py:338
#, python-format
msgid "volumes[] != string: %(volume)s"
msgstr "volumes[] != Zeichenkette: %(volume)s"

#~ msgid "Could not connect to \"%(uri)s\""
#~ msgstr "Keine Verbindung zu \"%(uri)s\""

#~ msgid "Domain required"
#~ msgstr "Domäne erwartet"

#~ msgid "Error retrieving network \"%(name)s\": %(error)s"
#~ msgstr "Fehler beim Lesen des Netzwerks %(name)s: %(error)s"

#~ msgid "File name or XML required"
#~ msgstr "Dateiname oder XML erwartet"

#~ msgid "File name required"
#~ msgstr "Dateiname erwartet"

#~ msgid "Frequency required"
#~ msgstr "Frequenz erwartet"

#~ msgid "Illegal URI: %(uri)s"
#~ msgstr "Fehlerhafte URI: %(uri)s"

#~ msgid "LDAP-uri for searching initial nodes"
#~ msgstr "LDAP-URI zum Suchen der initialen Knoten"

#~ msgid "No LDAP base in ldap/base."
#~ msgstr "Keine LDAP-Basis in ldap/base."

#~ msgid "No LDAP server in ldap/server/name."
#~ msgstr "Kein LDAP-Server in ldap/server/name."

#~ msgid "Pool name required"
#~ msgstr "Name für Speicherbereich erwartet"

#~ msgid "Skip reading data from LDAP"
#~ msgstr "Lesen der LDAP-Daten überspringen"

#~ msgid "State required"
#~ msgstr "Zustand erwartet"

#~ msgid "URI required"
#~ msgstr "URI erwartet"

#~ msgid "Volume name required"
#~ msgstr "Name für Speichermedium erwartet"

#~ msgid "Volume size required"
#~ msgstr "Größe des Speichermediums erwartet"

#~ msgid "ldap://HOST:PORT/BASE"
#~ msgstr "ldap://RECHNER:PORT/BASIS"

#~ msgid "usage: %%prog [options] ..."
#~ msgstr "Aufruf: %%prog [Optionen] ..."

#~ msgid "usage: %prog [options] command"
#~ msgstr "Aufruf: %prog [Optionen] Kommando"
